developer.bat

```diff
@@ -0,0 +1,17 @@
+IF NOT EXIST build mkdir build
+
+cd build
+
+cmake -DCMAKE_INSTALL_PREFIX=install -G "Visual Studio 16 2019" -DDEVELOPER=TRUE ..
```

Comment:
Should we hardcode VS version? I'm using 2022

Reply:
Can we ensure a VS generator without specifying the version? I'd gladly remove this flag, but if I do, I don't know what ensures that it will pick a visual studio generator.

Reply:
The only solution I've found to that is a bit complicated, but maybe worth using:
```bat
REM Detect the highest available Visual Studio version
for /f "tokens=*" %%i in ('"C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe" -latest -products * -requires Microsoft.Component.MSBuild -property installationVersion') do set VS_VERSION=%%i

REM The `VS_VERSION` will be something like 17.11.35312.102 ; keep only the major version
set VS_VERSION=%VS_VERSION:~0,2%

if %VS_VERSION%==16 (
    set GENERATOR="Visual Studio 16 2019"
) else if %VS_VERSION%==17 (
    set GENERATOR="Visual Studio 17 2022"
) else (
    echo No supported Visual Studio version found.
    exit /b 1
)

echo Using %GENERATOR%
```

Reply:
I see, we can give it a shot. One thing though, it should be `%programfiles(x86)%` instead of `C:\Program Files (x86)`